integration-cli/docker_cli_help_test.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
-	"os"
 	"os/exec"
 	"runtime"
 	"strings"
 	"testing"
 	"unicode"
+
+	"github.com/docker/docker/pkg/homedir"
 )
 
 func TestMainHelpWidth(t *testing.T) {
```

Comment:
hmmm, it should fail on master because the check looks for "/" and should Fatal if it sees it.
When you do `docker help` what do you see?
I get:
`
 -p, --pidfile="~var/run/docker.pid"    Path to use for daemon PID file
`